examples/spec/integration/schedule_workflow_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'workflows/hello_world_workflow'
+
+describe HelloWorldWorkflow do
+  subject { described_class }
+
+  before { allow(HelloWorldActivity).to receive(:execute!).and_call_original }
+
+  it 'can retrieve the workflow schedule via run id' do
+    workflow_id = 'schedule_test_wf'
+    cron_schedule = '*/5 * * * *'
+
+    begin
+      Temporal.terminate_workflow(workflow_id)
```

Comment:
nit: this should be part of a cleanup (e.g. after block) and not the test itself. Obvious reasons are — duplication, more noise and doing extra work when it might not be needed